lib/DDG/Spice/Pubmed/Search.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Pubmed::Search;
```

Comment:
Related to this, I think the tests are failing for this PR because there isn't an IA page for this yet. Is there any specific nomenclature I should use when creating that, or can it be called "Pubmed"?